google/cloud/aiplatform/training_jobs.py

```diff
@@ -2636,7 +2657,20 @@ def __init__(
             training_encryption_spec_key_name=training_encryption_spec_key_name,
             model_encryption_spec_key_name=model_encryption_spec_key_name,
         )
-        self._column_transformations = column_transformations
+        # user populated transformations
+        if column_transformations is not None and column_specs is not None:
+            _LOGGER.info(
+                "column_transformations and column_specs were both passed. column_transformations was used."
+            )
```

Comment:
Why not raise Error instead?

Reply:
I think that's more common, see what happens when gcs_source and bq_source are simultaneously passed to a Dataset.